From 3386c50d66759e4e6ddedabc178db3db33836aa6 Mon Sep 17 00:00:00 2001
From: "Jiri Slaby (SUSE)" <jirislaby@kernel.org>
Date: Fri, 3 Jul 2026 10:47:14 +0200
Subject: [PATCH 02288/15333] goldfish: remove unused gf_write_dma_addr()

The last user was removed in 2020 by commit c869eaa617e4 ("drivers:
staging: retire drivers/staging/goldfish").

Drop it.

Signed-off-by: Jiri Slaby (SUSE) <jirislaby@kernel.org>
Link: https://patch.msgid.link/20260703084717.176442-1-jirislaby@kernel.org
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 include/linux/goldfish.h | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/include/linux/goldfish.h b/include/linux/goldfish.h
index bcc17f95b906..40a059e03d78 100644
--- a/include/linux/goldfish.h
+++ b/include/linux/goldfish.h
@@ -26,15 +26,4 @@ static inline void gf_write_ptr(const void *ptr, void __iomem *portl,
 #endif
 }
 
-static inline void gf_write_dma_addr(const dma_addr_t addr,
-				     void __iomem *portl,
-				     void __iomem *porth)
-{
-	gf_iowrite32(lower_32_bits(addr), portl);
-#ifdef CONFIG_ARCH_DMA_ADDR_T_64BIT
-	gf_iowrite32(upper_32_bits(addr), porth);
-#endif
-}
-
-
 #endif /* __LINUX_GOLDFISH_H */
-- 
2.55.0

